src/roles/foreman_proxy/tasks/main.yaml

```diff
@@ -35,6 +34,20 @@
         PartOf=foreman.target
   notify: Restart Foreman Proxy
 
+- name: Create foreman-proxy.container.d folder
```

Comment:
With Puppet we also purged other entries. Do we still want to do that to disable features? Or go with an explicit `ensure => absent` equivalent on features?